index.html

```diff
@@ -42,5 +42,177 @@ <h1 class="name"> Hey there <br>
           <img src="images/Disabled.svg" alt="drop-down">
         </div>
     </section>
+    <section class="line-break">
+            <h2 class="works">My Recent <br> Works</h2>
+    </section>
+    
+      <section class="work-section">
+        <div  class="pictureholder-container">
+            <img src="Images 2/placeholder.svg" alt="placeholder" class="placeholder">
+            <div class="text-container">
+              <h5  class="header2">Multi-Post Stories <br> Gain+Glory</h5>
+              <ul class="text">
+                <li class="text-ruby">Ruby on Rails</li>
+                <li class="text-css">CSS</li>
+                <li class="text-java">JavaScript</li>
+                <li class="text-html">HTML</li>
+              </ul>
+              <div class="button1"><button class="button-text" type="button">See project </button></div>
```

Comment:
Great job on the button styles! However, there are a few adjustments needed to align with the design specifications:

- [ ] [Optional] Please add hover and active styles to the button to match the design. These styles will provide visual feedback to users when interacting with the button.
- [ ] [Optional] Instead of using fixed width and height for the button, consider using padding to achieve the desired size. This will allow for flexibility when the button text changes in the future, ensuring consistency across buttons with different text lengths.
![image](https://github.com/Wils-FOURR-speed/Portfolio-setup-and-mobile-first-project/assets/55840999/8c5e1391-f8a0-4901-afde-b146a0e7be75)

- [ ] [Optional] After the button you have margin-bottom 36px. But as per the design, you should have 24px. Please follow the design closely and update your UI.

By making these adjustments, you will ensure that the button styles are visually consistent with the design and provide a better user experience. Keep up the good work!